adi/ad3552r.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2024 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from decimal import Decimal
+
+from adi.attribute import attribute
+from adi.context_manager import context_manager
+from adi.rx_tx import tx
+
+
+class ad3552r(tx, context_manager):
+    """ AD3552R DAC """
+
+    _complex_data = False
+    _device_name = "AD3552R"
+    channels = []
```

Comment:
This is unused now. Is that expected? Not sure if this is just a name miss-match between channel and channel**s**

Reply:
Hi @tfcollins ,

fixed that, thanks.
Pushed a new version.